app/_locales/el/messages.json

```diff
@@ -1453,7 +1453,7 @@
     "message": "Γνωστή διεύθυνση συμβολαίου."
   },
   "knownTokenWarning": {
-    "message": "Αυτή η ενέργεια θα επεξεργαστεί tokens που είναι ήδη εισηγμένα στο πορτοφόλι σας, τα οποίαο μπορεί να χρησιμοποιηθούν για να σας κλέψουν στοιχεία. Αποδεχθείτε μόνο αν είστε σίγουροι ότι θέλετε να αλλάξετε αυτό που αντιπροσωπεύουν αυτά τα νομίσματα."
+    "message": "Αυτή η ενέργεια θα επεξεργαστεί tokens που είναι ήδη εισηγμένα στο πορτοφόλι σας, τα οποίαο μπορεί να χρησιμοποιηθούν για να σας κλέψουν στοιχεία. Αποδεχθείτε μόνο αν είστε σίγουροι ότι θέλετε να αλλάξετε αυτό που αντιπροσωπεύουν αυτά τα νομίσματα. Μάθετε περισσότερά γι'αυτό $1"
```

Comment:
This morning @adonesky1 met with our translations vendor and shared in [Slack](https://consensys.slack.com/archives/GTQAGKY5V/p1643210642285800):
> they are recommending that we not include translated strings until they've completed their proof-reading step

I think it's worth mentioning here that because this is a recommendation and not a strict rule, and because these translations are a verbatim copy and paste from the fakeTokenWarning.message translation, I think it's okay to add the translations for "Learn more about scams and security risks." in this PR.

Open to removing these translations if others feel otherwise.